### ISSUE TITLE
Enhancement - Case-Sensitivity settings option

### ISSUE THREAD

**Issue by JessieBurtonMiller** [jsleroy/compare-plugin#104]
Love this tool.  Please consider adding Case-Sensitivity setting option.  Beneficial to ignore case when comparing older source members that employ different case-styles.

**Comment by ti-jumbo** [comment]
Exatamente, essa função tornaria esse plugin praticamente possível de incorporá-lo definitivamente ao aplicativo

**Comment by bkl4114** [comment]
I agree, a case sensitivity option would be great. I use this tool to compare SQL a lot, and my SQL editor automatically changes all key words to upper case, so it's sometimes difficult to find the "real" differences. Thank you for this great tool!
![sql editing](https://user-images.githubusercontent.com/13295942/137748878-1f7c3f8d-6e09-4317-968d-14c339ce29c4.jpg)
